scripts/rspec/rspec.ps1

```diff
@@ -267,20 +267,31 @@ function AppendVbTestCase($ruleTestsFolder) {
     $existingClassText = Get-Content -Path "${ruleTestsFolder}\\${csClassName}Test.cs" -Raw
     $snippetText = Get-Content -Path "${RuleTemplateFolder}\\VbNetTestSnippet.cs" -Raw
 
-    $namespaceToken = "using CS = SonarAnalyzer.Rules.CSharp;"
+    $usingToken = "using SonarAnalyzer.Rules.CSharp;"
+    $csUsingToken = "using CS = SonarAnalyzer.Rules.CSharp;"
+    $vbNetUsingToken = "using VB = SonarAnalyzer.Rules.VisualBasic;"
+    $namespaceToken = "namespace SonarAnalyzer.UnitTest.Rules"
     $token = "    }"
-    $idx = $existingClassText.LastIndexOf($token)
-    $namespaceTokenIdx = $existingClassText.LastIndexOf($namespaceToken)
-    $newText = ""
+    $usingTokenIdx = $existingClassText.LastIndexOf($usingToken)
```

Comment:
Why is here `LastIndexOf`? There should be just one and using first seems like a better idea in case somebody uses that in a string literal of a embedded test case.